File: src/main/kotlin/org/jetbrains/plugins/template/common/path/RecourseScreen.java
package org.jetbrains.plugins.template.common.path;


import org.jetbrains.plugins.template.common.CommonString;
import org.jetbrains.plugins.template.common.FileNaming;

import java.io.File;

/// Путь до ресурсов Screen
public class RecourseScreen {

    private static final String baseResources = "template_screen";
    public static final  String screen        =
            baseResources + File.separator + CommonString.temp + FileNaming.screen;
    public static final  String wm            =
            baseResources + File.separator + CommonString.temp + FileNaming.wm;
    public static final  String route         =
            baseResources + File.separator + CommonString.temp + FileNaming.route;
    public static final  String di            =
            baseResources + File.separator + CommonString.di + File.separator + CommonString.temp +
            FileNaming.di;
}

File: src/main/kotlin/com/github/evoswork/plaginmwwm/common/path/RecourseScreen.kt
package org.jetbrains.plugins.template.common.path

import org.jetbrains.plugins.template.common.CommonString
import org.jetbrains.plugins.template.common.FileNaming
import java.io.File

/// Путь до ресурсов Screen
object RecourseScreen {
    private const val baseResources = "template_screen"
    val screen = baseResources + File.separator + CommonString.temp + FileNaming.screen
    val wm = baseResources + File.separator + CommonString.temp + FileNaming.wm
    val route = baseResources + File.separator + CommonString.temp + FileNaming.route
    val di = baseResources + File.separator + CommonString.di + File.separator + CommonString.temp +
            FileNaming.di
}